dist/mercury.js

```diff
@@ -6228,7 +6211,6 @@ var CustomExtractors = /*#__PURE__*/Object.freeze({
   WwwAlComExtractor: WwwAlComExtractor,
   WwwThepennyhoarderComExtractor: WwwThepennyhoarderComExtractor,
   WwwWesternjournalismComExtractor: WwwWesternjournalismComExtractor,
-  FusionNetExtractor: FusionNetExtractor,
```

Comment:
it looks like we're removing this?

when i do a search of the `FusionNetExtractor` extractor it shows up in the following files:
- `dist/generate-custom-parser.js.map`
- `dist/mercury.esm.js`
- `dist/mercury.js.map`

Reply:
wow computers are so so silly, thanks for documenting the commands!!